lib/custom_cops/spec_never_runs.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Getsafe
+  class SpecNeverRuns < RuboCop::Cop::Base
+    def on_new_investigation
+      path = processed_source.path
+      return unless path.include?('/spec/') && !path.end_with?('_spec.rb')
+
+      # Ignore non-spec files
+      return if path.match?(/spec\/(support|factories)\//) # Helpers and factories
+      return if path.match?(/(_examples|_helper)\.rb$/) # Shared examples
+
+      if processed_source.raw_source.include?('RSpec.describe')
+        add_global_offense("Spec file will never run because it is wrongly named")
+      end
```

Comment:
I wonder if this could be sped up, I think this has to read the whole file until the very end (and we have a lot of quite lengthy specs).

Reply:
Possibly. 🤔 But I think the impact is negligible:
- Every spec file is parsed, so `raw_source` is populated, regardless.
- `RSpec.describe` is typically near the start of the file, and `include?` should stop once it's found.